ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -410,11 +417,29 @@ func cliFlagsToCfnStackParams(context *cli.Context) *cloudformation.CfnStackPara
 	return cfnParams
 }
 
-// isIAMAcknowledged returrns true if the 'capability-iam' flag is set from CLI.
+// isIAMAcknowledged returns true if the 'capability-iam' flag is set from CLI.
 func isIAMAcknowledged(context *cli.Context) bool {
 	return context.Bool(command.CapabilityIAMFlag)
 }
 
+// returns true if customer specifies a custom instance role via 'role' flag.
```

Comment:
nit: lint would ask you to rewrite this as `// hasCustomRole returns` if it was a global method.